src/common/core/librarycore.js

```diff
@@ -216,6 +216,288 @@ define([
                 self.setProperty(overlay, to, fromOverlay);
             }
 
+            function isPathInSubTree(fullPath, subTreePath) {
```

Comment:
no:
full = '/a/be/dee'
subtree = '/a/b'

it would make the simple check true, however this is not a container of the full path...